charts/kueue/templates/kueueviz/backend-ingress.yaml

```
@@ -8,12 +8,15 @@ metadata:
     nginx.ingress.kubernetes.io/rewrite-target: /
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
 spec:
+  {{- if .Values.kueueViz.backend.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.kueueViz.backend.ingress.ingressClassName | default "ingress-nginx" }}
```

```suggestion
  ingressClassName: {{ .Values.kueueViz.backend.ingress.ingressClassName }}
```

Do we need default if we have if condition?

---

Or maybe we don't need if condition?